GliaWidgets/Configuration.swift

```diff
@@ -19,6 +21,8 @@ public enum Environment {
             // swiftlint:disable force_unwrapping
             return .custom(URL(string: "https://api.beta.salemove.com/")!)
             // swiftlint:enable force_unwrapping
+        case .custom(let url):
+            return .custom(url)
         }
     }
 }
```

Comment:
I wonder if we can add test that will ensure that Configuration can be created from URL.